detection_rules/packaging.py

```diff
@@ -399,8 +400,14 @@ def _generate_registry_package(self, save_dir):
         # shutil.copyfile(CHANGELOG_FILE, str(rules_dir.joinpath('CHANGELOG.json')))
 
         for rule in self.rules:
-            asset_path = rules_dir / f'{rule.id}.json'
-            asset_path.write_text(json.dumps(rule.get_asset(), indent=4, sort_keys=True), encoding="utf-8")
+            asset = rule.get_asset()
+            if self.historical:
```

Comment:
Nit: May be good to add a single doctoring explaining the purpose of this rule id logic

Reply:
Added.